### ISSUE TITLE
Could we share compressed model there?

### ISSUE THREAD

**Issue by ginger0106** [Tencent/PocketFlow#100]
I am trying to compress a lot models using pocketflow with different compressed ratios(0.4,0.6,0.8). But the GPU resources are limited and time is also limited, the process time  last too long. Could we share the compressed  models there? I will also share models there. Thanks.

**Comment by jiaxiang-wu** [comment]
Yes, we think it is a good idea to share compressed model in the community. Do you have any suggestion on where can we share such models (or how should we upload compressed models)?

**Comment by jiaxiang-wu** [comment]
Enhancement: share pre-trained compressed models.
